### PR TITLE
Eslint prettier

### PR DESCRIPTION
**Pull request by gurvanhenry**
Issue : #165 

Done:
- Add prettier in eslint config
- Apply lint/prettier on all existing app/* files.
- Add "yarn lint:fix" command  
- Minor update of README

Comments: 
 - I tried to find a prettier config that match the existing code base to make the minimum of file changes
 - prettier is automatically applied with "yarn lint" so I did not add script command like "yarn prettier"
 - husky already run "yarn lint" on pre-commit so I did not changed that
 - we can add a file .vscode/settings.json with the following content to format/fix file on save. (tell me if you want me to add it)

```
{
  "editor.codeActionsOnSave": {
    "source.fixAll.eslint": true
  },
}
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -98,9 +99,12 @@
     "babel-loader": "8.1.0",
     "babel-preset-expo": "8.2.3",
     "eslint": "6.7.2",
+    "eslint-config-prettier": "^6.12.0",
```
Comment (PierreBresson) [inline package.json:102]: Could you remove all the ^ from package.json? To make sure we all have the same version

**Review by PierreBresson — CHANGES_REQUESTED** [review]
Just a quick fix. If you can add `.vscode/settings.json` it would also be perfect 👌

**Comment by PierreBresson** [comment]
@gurvanhenry  Little question of me being curious, if you have time : how to apply changes to all the files without saving them one by one?

**Comment by gurvanhenry** [comment]
> @gurvanhenry Little question of me being curious, if you have time : how to apply changes to all the files without saving them one by one?

By running `yarn lint:fix` which include --fix parameter on eslint command.

**Comment by gurvanhenry** [comment]
> Just a quick fix. If you can add `.vscode/settings.json` it would also be perfect 👌

Done

File: package.json (hunk 1 above)
Comment (gurvanhenry) [reply package.json]: Done

**Comment by gurvanhenry** [comment]
I have synced my branch with master and re-applied lint on the file `NoEmission.story.tsx` to fix the CI

**Review by PierreBresson — APPROVED** [review]

**Comment by PierreBresson** [comment]
@gurvanhenry top, thanks!

**Pull request merged**